infra/deploy/spoke/variables.tf

```diff
@@ -62,3 +62,15 @@ variable "service_bus_firewall" {
 variable "appsettings_key_value" {
   type = map(string)
 }
+
```

Comment:
bør vi ha en prefix som indiktere at dette er appsettings?

appsettings_{name}

mulig det blir litt verbose?

Kanskje det kunne lønnet seg å lage et appsettings objekt?

object({
platform_resource_registry_endpoint = string
platform_register_endpoint = string
platform_sbl_bridge_endpoint = string


})

Reply:
Object it is :)